shared/chat/inbox/index.desktop.tsx

```diff
@@ -282,7 +293,7 @@ class Inbox extends React.Component<T.Props, State> {
   private listChild = ({index, style}) => this.itemRenderer(index, style)
 
   private onDragOver = e => {
-    if (this.scrollDiv.current) {
+    if (this.scrollDiv.current && e.dataTransfer.types.length > 0 && e.dataTransfer.types[0] === dragKey) {
```

Comment:
don't actually need the length check right

Reply:
Thought it couldn't hurt